src/templates/homepage/InfocolsSection.tsx

```diff
@@ -0,0 +1,163 @@
+import _ from "lodash"
+import { ForwardedRef, forwardRef } from "react"
+
+import editorStyles from "styles/isomer-cms/pages/Editor.module.scss"
+
+import { getClassNames } from "templates/utils/stylingUtils"
+
+import { InfocolsSection } from "types/homepage"
+
+interface TemplateTextCardsSectionProps extends InfocolsSection {
+  sectionIndex: number
+}
+
+const TemplateInfocolsSection = forwardRef<
+  HTMLDivElement,
+  TemplateTextCardsSectionProps
+>(
+  (
+    {
+      title,
+      subtitle,
+      linktext,
+      url,
+      infoboxes,
+      sectionIndex,
+    }: TemplateTextCardsSectionProps,
+    ref: ForwardedRef<HTMLDivElement>
+  ) => {
+    const getInfoBox = (infoboxTitle: string, infoboxDescription?: string) => {
```

Comment:
follows same structure as in templates. infobox is an includes file in templates which has been mimicked here with a extracted function

Reply:
not a real issue, just curious why you chose to use a function over a component

Reply:
Oh ps, I was ok with the function, just curious on the reason!!! Don't feel pressured to do this

Reply:
done